.github/workflows/scripts/unix_test.sh

```diff
@@ -20,6 +20,7 @@ fi
 python3 -m pip install dist/*.whl
 if [ -z "$GPU_TEST" ]; then
     python3 -m pip install -r requirements_test.txt
+    python3 -m pip install torch
```

Comment:
is this change related to the PR?

Reply:
Yes, torch is removed from `requirements_text.txt` since it uses a different version for GPU build.